database/src/main/resources/db/migration/V1_169__change_address_post_code.sql

```diff
@@ -0,0 +1,65 @@
+ALTER TYPE address
+    ADD ATTRIBUTE "foreign_postal_code" VARCHAR(10);
+
+-- Create backup for "transport-service" table
+CREATE TABLE "transport-service_backup" AS TABLE "transport-service";
```

Comment:
Was this table intended to be saved or should it be dropped in the PR?

Reply:
This is for backup purposes. I'm a bit afraid to update all address data.